prime-router/src/main/kotlin/azure/ReportFunction.kt

```diff
@@ -0,0 +1,309 @@
+package gov.cdc.prime.router.azure
```

Comment:
The report function is combination of the old Validate and Translate functions. By doing both operations in one-step, it is now possible to do a much more complete validation which includes translation errors.

Reply:
Agree. Can't do everything in a single pull request.